docs/directory-api-reference/overview.mdx

```diff
@@ -1,190 +1,29 @@
 ---
 sidebar_label: Overview
-title: Directory Guide
-description: How to use the Aserto Directory GraphQL API
+title: Directory API
+description: How to use the Aserto Directory API
 ---
 
 # Overview
 
-## Authentication
+Using Aserto's Directory API you can access the manifest, retrieve and update objects in the directory and work with relations.
 
-Two HTTP headers must be included in all requests to the directory API:
+## Checks
+* Check Permission - Check if a subject has permission on the specified object.
+* Check Relation - Check it a subject has requrested relation on the specific object.
 
-* `Authorization` - an API key (`basic <KEY>`) or OAuth2.0 token (`bearer <TOKEN>`).
+## Manifest
+* Get Manifest - Retrieve the manifest
```

Comment:
these are not quite operational yet?  (at least not deployed to prod yet)

Reply:
Removed for now